functions/v2/pubsub/main.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import base64
+
+
+# [START functions_cloudevent_pubsub]
+# Triggered from a message on a Cloud Pub/Sub topic.
+def subscribe(cloudevent):
+    # Print out the data from Pub/Sub, to prove that it worked
+    print(base64.b64decode(cloudevent.data["message"]["data"]))
```

Comment:
Just looking at the Go version of this. Nice to see how concisely this is represented in Python. I compared this to what I understand from https://github.com/GoogleCloudPlatform/functions-framework-python and this makes sense to me.